server/routes/feedback.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../models");
+
+router.get("/", async (req, res) => {
+    const feedbacklist = await db.feedback.findAll();
+    if (!feedbacklist) {
+        res.status(404).send("something went wrong");
```

Comment:
Bei einem Serverfehler sendest du `500`, wenn es geht sogar präziser (also `502` bei bsp. Bad Gateway).
Anfragen Fehler sind `400`er Fehler. `404` allerdings nur, wenn eine explizit angegebene Ressource nicht gefunden werden kann.

Reply:
Gilt für alle Endpoints

Reply:
mit !feedbacklist wird nur überprüft ob die datenbank anfrage daten zurückgeliefert hat oder nicht (also keine entsprechenden daten gefunden hat)

damit würde ich sagen trifft 404 hier am besten zu da es kein serverfehler an sich ist

ref: https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.4

Reply:
Ja, wenn du davon ausgehst, dass es keinen Fehler bei der Abfrage zur DB gibt. Was aber nicht zwingend der Fall sein muss, wenn z.B. ich mich dämlich anstelle und die DB in dem Moment nicht erreichbar ist.

Reply:
das würde aber hierbei auch nicht abgefangen werden, weil in dem fall dann ein error geworfen wird und das backend terminiert

error catching wollte ich später noch machen aber jetzt zum ersten testen hab ich das erstmal raus gelassen ums übersichtlicher zu halten

Reply:
Alles klar, dann ist ja ok.